test/night-render.hpp

```diff
@@ -140,7 +139,8 @@ struct night_render
 
         glBindFramebuffer(GL_FRAMEBUFFER, lum_fb);
         glBindTexture(GL_TEXTURE_2D_MULTISAMPLE, lum_tex);
-        glTexImage2DMultisample(GL_TEXTURE_2D_MULTISAMPLE, nsamples, GL_RGBA8, dim[0], dim[1], false);
+        // glTexImage2DMultisample(GL_TEXTURE_2D_MULTISAMPLE, nsamples, GL_RGBA8, dim[0], dim[1], false);
+        glTexImage2DMultisample(GL_TEXTURE_2D_MULTISAMPLE, nsamples, GL_RGBA8, dim[0], dim[1], true);
```

Comment:
What is this? Why do we need fixed sample locations?

Reply:
When I run the original code, the program breaks down because
` Assertion `glGetError() == GL_NO_ERROR' failed`
after some `glBindBuffer()`

Then I checked the framebuffers by 
`glCheckFramebufferStatus`
It returns `GL_FRAMEBUFFER_INCOMPLETE_MULTISAMPLE` instead of `GL_FRAMEBUFFER_COMPLETE`
Which could mean that
```if the value of GL_TEXTURE_FIXED_SAMPLE_LOCATIONS is not the same for all attached textures; or, if the attached images are a mix of renderbuffers and textures, the value of GL_TEXTURE_FIXED_SAMPLE_LOCATIONS is not GL_TRUE for all attached textures.```

https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glCheckFramebufferStatus.xhtml

So I changed it to a fixed location.

Is there a problem if I did this? Or what else I can do to fix this?